policy-design-lab/pdl-frontend
# House Outlay Projection Front-end Implementation

This PR adds the House Outlay Projection page, addressing #348 with #330. It also hides the issue brief in the 1.6.0 release (noted in the CHANGELOG). The development site has been updated accordingly.

## How to Test

1. Navigate to the Proposal Analysis Tab
- Go to https://policydesignlab-dev.ncsa.illinois.edu/policy-lab
- Click on the second tab, "Proposal Analysis".
- You should be directed to the House Outlay Projection page. 
<img width="1107" alt="Screenshot 2025-02-24 at 4 15 02 PM" src="https://github.com/user-attachments/assets/9ea8af43-d6d3-4322-8cb2-0f94b7868c78" />

2. Verify the Sidebar Menu
- You should see a three-level menu on the left side of the page.
- Only the last menu item should be clickable and highlighted, indicating the current page level. 
<img width="391" alt="Screenshot 2025-02-24 at 4 16 38 PM" src="https://github.com/user-attachments/assets/f542139c-adc5-4db4-a426-626735eadfce" />

3. Test Map Selections
- Select different practices on the map.
- The map tooltip should update to reflect your selections.
<img width="797" alt="Screenshot 2025-02-24 at 4 17 33 PM" src="https://github.com/user-attachments/assets/b78680a2-dd17-4487-8fa1-3ea9899d7a19" />

4. Test 'All Practices' Selection
- Select the "All Practices" option from the practice selector OR remove all selected practices.
- The map should reset to its initial state, showing the total values.

5. Verify Table Updates
- In steps 3 and 4, observe the table at the bottom of the page.
- The table should update in sync with the map selections.
<img width="1139" alt="Screenshot 2025-02-24 at 4 20 37 PM" src="https://github.com/user-attachments/assets/89652d48-9454-440b-b990-5e7a06215aee" />

6. Export Table to CSV
- Click the "Export Table to CSV" button.
- The downloaded CSV file should contain all table data.
7. Verify Issue Brief Visibility
- Navigate to https://policydesignlab-dev.ncsa.illinois.edu/issues-whitepapers. You should see only ONE issue brief listed.
